dbt_coves/tasks/setup/ssh.py

```diff
@@ -36,7 +36,7 @@ def run(cls) -> int:
         ssh_configured = False
         ssh_keys_dir = "~/.ssh/"
         ssh_keys_dir_abs = os.path.abspath(Path(ssh_keys_dir).expanduser())
-        key_path_abs = f"{ssh_keys_dir_abs}/id_ecdsa"
+        key_path_abs = f"{ssh_keys_dir_abs}/id_dbtcoves"
```

Comment:
I would just use standard names, why switching to `id_dbtcoves`?

Reply:
What do you mean by standard? id_rsa, id_ecdsa, etc? If we do that we should add a questionary prompting the user to choose which type is providing/generating

We chose defaulting to ECDSA type as it's the latest protocol, and if the user provides a different one the file name wouldn't be the proper one